src/pages/Dashboard/AdCenter/index.tsx

```diff
@@ -0,0 +1,30 @@
+import Ad from '../../../features/Ad';
+import PageHeader from '../../../features/PageHeader';
+import * as constant from './contant';
+
+const Box = (): JSX.Element => (
+    <div className='border rounded-sm w-36 p-1.5 border-gray-300 flex flex-row items-center bg-gray-300 text-white'>
+      <span className="material-symbols-outlined">expand_more</span>
+      <span className='flex-1'>User</span>
+      <span className="material-symbols-outlined">account_box</span>
+    </div>
```

Comment:
The amount of space before each html element is too big. The div should be at 2 spaces from the side of the page and the span at 4 spaces. This is the reason one of the checks is failing.